Plandomizer.py

```diff
@@ -1031,12 +1035,18 @@ def pattern_dict_items(pattern_dict, itempool=None, used_items=None):
                 valid_items = value.item
             if not valid_items and used_items is None:
                 continue
-            elif not valid_items:
+            elif not valid_items and used_items is not None:
+                limited_items = ['Weird Egg', '#AdultTrade', '#Bottle']
+                for item in value.item:
```

Comment:
Modifying a list while iterating makes me nervous. Might as well just construct a new list like `[v for v in value.item if v not in limited_items and v not in ...]`. (Unless you can prove to me this is safe and quite a bit more efficient...)